src/main/java/Events/RandomEvent.java

```diff
@@ -0,0 +1,90 @@
+package Events;
+
+import Bot.DiscordBot;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+import javax.annotation.Nonnull;
+import java.awt.*;
+import java.util.Random;
+
+public class RandomEvent extends ListenerAdapter {
+    private EmbedBuilder card = new EmbedBuilder();
+
+    @Override
+    public void onGuildMessageReceived(@Nonnull GuildMessageReceivedEvent event) {
+        card.setColor(Color.BLUE);
+
+        String message = event.getMessage().getContentRaw();
```

Comment:
```suggestion
        String[] message = event.getMessage().getContentRaw();
```

instead of having a String variable and then implementing it into an array

Reply:
Could you please pull latest master into your branch for updates on DiscordBot.java